ui/FluidStyle.qml

```diff
@@ -17,4 +18,19 @@ QtObject {
 
     readonly property color iconColorLight: Qt.rgba(0,0,0,0.54)
     readonly property color iconColorDark: Qt.rgba(1,1,1)
+
+    readonly property font subheadingFont: Qt.font({ family: "Roboto",
+                                                     pointSize: deviceOrDesktop(16, 15) })
+
+    readonly property font body1Font: Qt.font({ family: "Roboto",
+                                                     pointSize: deviceOrDesktop(14, 13) })
+
+    /* Control fonts that don't fit into the standard font styles */
+
+    readonly property font subheaderFont: Qt.font({ family: "Roboto", weight: Font.DemiBold,
+                                                    pointSize: 14 })
+
+    function deviceOrDesktop(device, desktop) {
+        return Device.isMobile ? device : desktop
```

Comment:
Mostly because it abstracts away how device vs desktop is determined. Right now it's based on Device.isMobile, but if we decided it would be more correct to do it based on form factor, we could do that as well.

I can get rid of the function and just use Device.isMobile if you'd prefer.